slam-ws/src/nav_goal/src/nav_goal/movebase_client.py

```diff
@@ -0,0 +1,37 @@
+import rospy
+import actionlib
+from move_base_msgs.msg import MoveBaseAction, MoveBaseGoal
+from tf.transformations import quaternion_from_euler
+
+def movebase_client(x, y, yaw):
+  # Create an action client called "move_base" with action definition file "MoveBaseAction"
+  client = actionlib.SimpleActionClient('move_base', MoveBaseAction)
+  # Waits until the action server has started up and started listening for goals.
+  client.wait_for_server()
+
+  # Creates a new goal with the MoveBaseGoal constructor
+  goal = MoveBaseGoal()
+
+  goal.target_pose.header.frame_id = "base_link"
+  goal.target_pose.header.stamp = rospy.Time.now()
+
+  # Set goal position
+  goal.target_pose.pose.position.x = x;
+  goal.target_pose.pose.position.y = y;
+
+  # Convert yaw to quaternion and set pose
+  q = quaternion_from_euler(0, 0, yaw)
+  goal.target_pose.pose.orientation.x = q[0];
+  goal.target_pose.pose.orientation.y = q[1];
+  goal.target_pose.pose.orientation.z = q[2];
+  goal.target_pose.pose.orientation.w = q[3];
+
+  client.send_goal(goal)
+  wait = client.wait_for_result()
```

Comment:
Honestly I think the variable names are self explanatory ....but everything before this is so nicely commented it makes me feel like a comment for wait_for _result is missing something like waits until server completes execution reflected in sim or something along those lines what do u guys think @Federico-PizarroBejarano @erickmu1

Reply:
Yeah you're right, will fix